src/main/lrsql/system/lrs.clj

```diff
@@ -328,4 +328,10 @@
       (jdbc/with-transaction [tx conn]
         (auth-cmd/delete-credential! backend tx cred-in)
         {:api-key    api-key
-         :secret-key secret-key}))))
+         :secret-key secret-key})))
+  adp/AdminStatusProvider
```

Comment:
Should have an extra space

Reply:
https://github.com/yetanalytics/lrsql/pull/282/commits/44b6d23089d8bcde98d2cc2832024436bf8139ba